src/platform/generic.ts

```diff
@@ -0,0 +1,73 @@
+import { NextApiRequest } from "next";
+import { RequestReport } from "../logger";
+import { EndpointType } from "../shared";
+import type PlatformConfigurator from "./base";
+
+export default class GenericConfig implements PlatformConfigurator {
```

Comment:
Please add a comment saying that this is for everything that's not Vercel or Netlify